5calls/app/src/main/java/org/a5calls/android/a5calls/net/FiveCallsApi.java

```diff
@@ -345,6 +347,31 @@ public Map<String, String> getHeaders() throws AuthFailureError {
         mRequestQueue.add(request);
     }
 
+    public void reportSearch(String searchTerm) {
```

Comment:
Also consider not reporting the search on debug builds.

Reply:
👍